packages/database/src/tests/changeHandlers/AnalyticsRefresher.test.js

```diff
@@ -190,4 +193,77 @@ describe('AnalyticsRefresher', () => {
     });
     await assertAnalyticsMatch(updatedAnalytics);
   });
+
+  it('refreshes analytics table if surveys are updated', async () => {
+    // Update an answer, make sure analytic is updated
```

Comment:
```suggestion
    // Update a survey, make sure analytic is updated
```

similarly for the other test cases